MetaCocktailsSwiftData/Model/Components/Glassware.swift

```diff
@@ -15,4 +15,7 @@ enum Glassware: String, Codable {
     case martini        = "Martini"
     case flute          = "Flute"
     case smallHighball  = "Small Chilled Highball"
+    
+    case stirred        = "Stirred"
+    case shaken         = "Shaken"
```

Comment:
These shouldn't be in glassware.

D you think we should add a "method" to the cocktail model? for shaken / stirred / built etc?